cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -116,6 +55,24 @@
     .o-expandable_header-right {
         width: 27%;
     }
+
+    .respond-to-max( @bp-xs-max, {
+        .swipe {
+          overflow: hidden;
+          visibility: hidden;
```

Comment:
Is the visibility over-written by the JS? If so, do we have a fallback for non-js situations?